samples/docker/README.md

```diff
@@ -4,6 +4,48 @@
 
 The following instructions detail how to build and run the FHIR Server in Docker on Linux.
 
+## Use CI image
+
+If it is not desirable to clone this repository and build locally an image of the most recent CI build is available from the Azure Container Repository HealthPlatformRegistry. 
+
+Using docker-compose this image can be started with the following steps:
+1. Open a terminal window.
+2. Login to the Azure Container Registry 'healthplatformregistry' (https://docs.microsoft.com/en-us/azure/container-registry/container-registry-get-started-docker-cli)
```

Comment:
Is login step still required (with anonymous pull enabled registry)?

Reply:
Good catch, removed